lib/src/base-styles/_nx-variables.scss

```diff
@@ -42,6 +42,7 @@
   --nx-border-radius: 6px;
 
   --nx-border-default: 1px solid var(--nx-color-border);
+  --nx-border-secondary: 1px solid var(--nx-color-border-secondary);
   --nx-border-subsection: 1px solid var(--nx-color-subsection-border);
```

Comment:
Looks like we're replacing `--nx-border-subsection` with` --nx-border-secondary.` Do we even need `--nx-border-subsection` anymore?

Reply:
> Do we even need --nx-border-subsection anymore?

For backwards compatibility, yes.  But it should be defined as just an alias to `--nx-border-secondary` and should be documented as deprecated.  Same with `--nx-color-subsection-border`.

Reply:
> > Do we even need --nx-border-subsection anymore?
> 
> For backwards compatibility, yes. But it should be defined as just an alias to `--nx-border-secondary` and should be documented as deprecated. Same with `--nx-color-subsection-border`.

Subsection borders are deprecated here: https://github.com/sonatype/sonatype-react-shared-components/pull/444/commits/0832a751def52ca8ab1bf66b5ddf5bcfd85df38d and the doc entry was removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/444/commits/82dc27415b4429cf0d7336cfd3ea0a0e507da291